restrict_download_region/restrict_region.py

```diff
@@ -26,46 +29,24 @@ def handler(event: dict, context: dict):
         if not BUCKET_NAME:
             raise ValueError("BUCKET_NAME must be defined in enviroment variables")
 
-        region_ip_prefixes = get_ip_prefixes_for_region() \
```

Comment:
This was an artifact from back when I was trying to handle multiple bucket's policies. It's been refactored to be called by `process_ip_restrict_policy`, which already handles logic when `custom_resource_type == Delete`